solutions/src/main/java/ru/sberbank/school/task13/cacheproxy/CacheSerializatorManager.java

```diff
@@ -0,0 +1,76 @@
+package ru.sberbank.school.task13.cacheproxy;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.esotericsoftware.kryo.KryoException;
+import com.esotericsoftware.kryo.io.Input;
+import com.esotericsoftware.kryo.io.Output;
+import lombok.NonNull;
+
+import java.io.*;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import java.util.zip.ZipOutputStream;
+
+public class CacheSerializatorManager {
+    private Kryo kryo = new Kryo();
+
+    public void writeMethodToFile(File file, Object methodRes) throws FileNotFoundException {
+        try (FileOutputStream fos = new FileOutputStream(file);
+             Output out = new Output(fos)) {
```

Comment:
&#39;try resource&#39; has incorrect indentation level 13, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-2015986192>*